_dasldemos/proof-troubleshooting.md

```diff
@@ -0,0 +1,104 @@
+---
+title: PROOF Troubleshooting
+main_authors: tefirman
+primary_reviewers: laderast
+---
+
+For users that have a well-tested and established WDL workflow, the [PROOF How-To documentation](/dasldemos/proof-how-to/) should provide enough guidance to help a user through a typical process of job submission. However, while developing a new and untested WDL workflow, it's common to run into all sorts of issues, some as simple as typos, others as complex as entire software environments. The goal of this guide is to teach users where to find (and how to interpret) the more advanced features and behaviors of PROOF in order to debug their custom WDL scripts and get their research up and running. 
+
+## Common PROOF Server Issues
+
+### Make sure your PROOF server is up and running
+
+- By default, PROOF servers run for seven days before they shut down automatically. If submitting/validating a workflow produces an error message like “Your server is not up!”, you’ll need to start up another PROOF server via [the “PROOF Server” tab](/dasldemos/proof-how-to#starting-a-proof-server). Analysis jobs in progress will not be affected by the restart and all previous job history will be saved.
+- If you just started your PROOF server, and received this message, the server may need additional time before it can communicate with the app, wait 1-2 minutes and try again.  
+- If the screen is grayed out and you’re unable to press buttons at all, your browser has timed out. You’ll need to refresh the page and log in again.
+
+### Try stopping and restarting your PROOF server
+
+- Occasionally, our product team will push updates to the underlying repositories of PROOF to fix issues and add functionalities. However, only servers created _after_ the update will get that added/fixed functionality.
+- If your server has been running for a few days, it is definitely worth [stopping and restarting](/dasldemos/proof-how-to/#stopping-a-proof-server) your PROOF server. If any recent updates have been pushed, your new server will pick them up and hopefully that will fix your issue. And even if nothing's been updated, always worth trying the old standby of "turning it off and on again."
+
+## Workflow Level Issues
```

Comment:
Can also add that users should 'Reset' and re-uploaded file after every change made to WDL or JSON.

Reply:
Great suggestion, added to the "Execution Issues" section!